src/app.js

```diff
@@ -56,15 +56,16 @@ app.get('/my-started-game', async (req, res) => {
   try {
     const { playerId } = req.query
 
-    if (!playerId) return res.status(400).send({ message: 'Field playerId is required!' })
+    if (playerId === 'null') return res.status(400).send({ message: 'Field playerId is required!' })
```

Comment:
Why did we go from `if (!playerId)` to `if (playerId === 'null')` ?

Reply:
Beause when using UI and there is no playerId it come from req.query as a string so "!playerId" not recognize the string as null and we can never reach this condition

Reply:
If this is the case I think it is safer to add both checks here:
`!playedId || playedId === 'null'`

Also worth checking is why do we have a string `'null'` and not plain `null`. Can this be due to the way the request is made?

Reply:
Actually, we discussed this problem with Slaven, so we both decided to leave it as it was for now, I mean this way "!playerId", and make backlog as a bug for later because the problem is in the querying of the URL.